### DISCUSSION TITLE
About the outline of jupyter notebook:

### DISCUSSION THREAD

**Discussion by YexiangYan** [microsoft/vscode-jupyter#10176] [New Ideas]
Thanks for such a great extension for jupyter notebook. I think if you can add two functions in the notebook directory, it will improve the user experience:
(1) All markdown cells will be displayed in the outline. Can only the headings marked with # be displayed and indented by level? This will make the outline clearer.
(2) Is it possible to increase the title number according to the title's level? For example 1, 1.1, 1.1.1 and so on.
Thank you for your efforts.

**Comment by rchiodo** [comment]
Markdown outline is actually implemented in VS code core. I suggest adding an issue there:
https://github.com/microsoft/vscode/issues

**Comment by rchiodo** [comment]
See these two issues which sound like what you're asking for:
https://github.com/microsoft/vscode/issues/147007
https://github.com/microsoft/vscode/issues/147008
